### PR TITLE
feat(release): add changelog entry due to cli release

### LINKED ISSUE #1937: add changelog entry due to changes on go-scalingo project
We need to make a new changelog entry after the released of the `go-scalingo` project after this issue was done: https://github.com/Scalingo/go-scalingo/issues/301

### PR DESCRIPTION
**Pull request by ipfaze**
fix #1937

### CODE REVIEW

File: src/changelog/cli/_posts/2023-02-07-1-28-0.md
Comment (EtienneM) [inline src/changelog/cli/_posts/2023-02-07-1-28-0.md:2]: This date and the date in the file name is not the same. Can you update?

File: src/changelog/cli/_posts/2023-02-06-1-28-0.md
Comment (EtienneM) [inline src/changelog/cli/_posts/2023-02-06-1-28-0.md:13]: Why a CLI release with such a limited changelog? Is it really useful to release a new version for the CLI? Moreover I expect the changelog to be more understandable by customers. Maybe this upgrade adds a useful feature for our customers?

Comment (SCedricThomas) [reply src/changelog/cli/_posts/2023-02-06-1-28-0.md:13]: This dependency upgrade allows the CLI to have a better interpretation of the new implemented 429 too many requests errors. Maybe we can wait for the next feature to be merged for the CLI https://github.com/Scalingo/cli/pull/882  ?

Comment (EtienneM) [reply src/changelog/cli/_posts/2023-02-06-1-28-0.md:13]: If that's the only change I would release only a patch version of the CLI and the changelog would be something like:

```
* chore(deps): better management of "429 Too Many Requests" errors (upgrade github.com/Scalingo/go-scalingo/v6.2.0)
```

File: src/changelog/cli/_posts/2023-02-06-1-28-0.md
Comment (EtienneM) [inline src/changelog/cli/_posts/2023-02-06-1-28-0.md:2]: Don't forget to update the date here and in the filename before merging so that it matches the date of release

**Review by EtienneM — APPROVED** [review]

**Pull request merged**
